js/interface.js

```diff
@@ -2802,6 +2807,20 @@ $('form').validator({
         return true;
       }
 
+      return false;
+    },
+    'validation-authentication-key': function($el) {
+      var authKey = $el.val().replace(/\\n/g, '\n');
+      var authKeyRegExp = /^(-----BEGIN\sPRIVATE\sKEY-----\n)+(.|\n)+(\n-----END\sPRIVATE\sKEY-----)$/;
```

Comment:
Why is this `+` necessary? We don't need more than one `-----BEGIN\sPRIVATE\sKEY-----\n`.

```suggestion
      var authKeyRegExp = /^(-----BEGIN\sPRIVATE\sKEY-----\n)(.|\n)+(\n-----END\sPRIVATE\sKEY-----)$/;
```